packages/css-framework/src/components/_scrollable-nav.scss

```diff
@@ -28,8 +28,13 @@ $btn-focus-width: 0.2rem;
   border-top: 4px solid transparent;
   border-bottom: 4px solid transparent;
   cursor: pointer;
+  position: relative;
 
-  &:focus {
+  &:hover {
+    border-bottom: 4px solid color(primary, 600);
```

Comment:
There wasn't a border in the design as far as was aware but I did think it needed 'something'.

Reply:
Yeah, when fiddling with it, that interaction felt the best. Having a fresh look at it in context helped.